.github/workflows/release.yaml

```
@@ -0,0 +1,37 @@
+name: Release
+on:
+  workflow_dispatch:
+
+jobs:
+  build-and-test:
+    uses: ./.github/workflows/build-and-test.yaml
+    secrets:
+      SIEMENS_NPM_TOKEN: ${{ secrets.SIEMENS_NPM_TOKEN }}
+      SIEMENS_NPM_USER: ${{ secrets.SIEMENS_NPM_USER }}
+
+  publish:
+    runs-on: ubuntu-22.04
+    needs:
+      - build-and-test
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.ELEMENT_BOT_GITHUB_TOKEN }}
+          fetch-depth: 0 # semantic-release needs this
+      - uses: actions/setup-node@v4
+        with:
+          node-version: lts/jod
+          cache: 'npm'
+      - uses: actions/download-artifact@v4
+        with:
+          name: dist
+          path: dist
+      - run: npm ci --prefer-offline --no-audit
+      - run: npx semantic-release
+        env:
+          GIT_AUTHOR_NAME: 'Siemens Element bot'
```

Shouldn't we go all-caps here?

```suggestion
          GIT_AUTHOR_NAME: 'Siemens Element Bot'
```